src/Convert/Convert.php

```diff
@@ -129,6 +129,42 @@ public function toDM($format = ConvertInterface::DEFAULT_DM_FORMAT)
         return $this->toDecimalMinutes($format);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function toDegreeDecimalMinutes($format = ConvertInterface::DEFAULT_DDM_FORMAT)
+    {
+        $latitude  = $this->parseCoordinate($this->coordinates->getLatitude());
+        $longitude = $this->parseCoordinate($this->coordinates->getLongitude());
+
+        $decimalPrecisionFormat = sprintf('%%0.%df', ConvertInterface::DEGREE_DECIMAL_MINUTES_PRECISION);
+        $latitude['decimalMinutes']  = sprintf($decimalPrecisionFormat, $latitude['decimalMinutes']);
+        $longitude['decimalMinutes'] = sprintf($decimalPrecisionFormat, $longitude['decimalMinutes']);
+
+        return strtr($format, [
+            ConvertInterface::LATITUDE_SIGN             => $latitude['positive'] ? '' : '-',
+            ConvertInterface::LATITUDE_DIRECTION        => $latitude['positive'] ? 'N' : 'S',
+            ConvertInterface::LATITUDE_DEGREES          => $latitude['degrees'],
+            ConvertInterface::LATITUDE_DECIMAL_MINUTES  => $latitude['decimalMinutes'],
+            ConvertInterface::LONGITUDE_SIGN            => $longitude['positive'] ? '' : '-',
+            ConvertInterface::LONGITUDE_DIRECTION       => $longitude['positive'] ? 'E' : 'W',
+            ConvertInterface::LONGITUDE_DEGREES         => $longitude['degrees'],
+            ConvertInterface::LONGITUDE_DECIMAL_MINUTES => $longitude['decimalMinutes'],
+        ]);
+    }
+
+    /**
+     * Alias of toDegreeDecimalMinutes function.
```

Comment:
We dont need an alias, do we? It serves no real benefit.

Reply:
I added it because there are already 2 aliases :  
- https://github.com/thephpleague/geotools/blob/master/src/Convert/Convert.php#L95
- https://github.com/thephpleague/geotools/blob/master/src/Convert/Convert.php#L127

Reply:
I understand. I would prefer to deprecate those aliases instead.

Reply:
Done